nohup: ignoring input
ESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

<<<<<<< HEAD
Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

i ---> 0
No se encontraron procesos con el nombre 'java'.
Procesos con el nombre 'chrome' detenidos correctamente.
Se eliminaron los archivos bmp.log y server.log
i ---> 1
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 2
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 3
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 4
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 5
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 6
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 7
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 8
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 9
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 10
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 11
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 12
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 13
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 14
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 15
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 16
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 17
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 18
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 19
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 20
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 21
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 22
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 23
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 24
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 25
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 26
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 27
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 28
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 29
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 30
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 31
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 32
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 33
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 34
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 35
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 36
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 37
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 38
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 39
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 40
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 41
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 42
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 43
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 44
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 45
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 46
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 47
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 48
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 49
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 50
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 51
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 52
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 53
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 54
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 55
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 56
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 57
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 58
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 59
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 60
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 61
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 62
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 63
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 64
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 65
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 66
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 67
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 68
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 69
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 70
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 71
No se encontraron procesos con el nombre 'java'.
No se encontraron procesos con el nombre 'chrome'.
i ---> 72
No se encontraron procesos con el nombre 'java'.
ESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/orcid/0_json
Registros originales: 7053
Traceback (most recent call last):
  File "/home/nahumfg/RenacytScraper/y3-Orcid.py", line 118, in <module>
    df_ejecuciones_base['cti_vitae'] = df_ejecuciones_base['cti_vitae'].str.replace('_Error', '')
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 181, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/nahumfg/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 235, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!. Did you mean: 'std'?
=======
Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/renacyt/perfil/0_json
Registros originales: 7211

Registros filtrados: 624
Registros a ejecutar: 3
Valor de max_i: 7529
Se inicia el servidor en Mac o Linux
Usando el chromedriver_path: /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver
7529. https://ctivitae.concytec.gob.pe/buscador-ui/#/ficha/ficha-renacyt?idInvestigador=204506
Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/renacyt/perfil/0_json
Se inicia el servidor en Mac o Linux
Usando el chromedriver_path: /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver
7530. https://ctivitae.concytec.gob.pe/buscador-ui/#/ficha/ficha-renacyt?idInvestigador=204525
Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/renacyt/perfil/0_json
Se inicia el servidor en Mac o Linux
Usando el chromedriver_path: /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver
7531. https://ctivitae.concytec.gob.pe/buscador-ui/#/ficha/ficha-renacyt?idInvestigador=204566
Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/renacyt/perfil/0_json
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/renacyt/perfil/0_json
Registros originales: 7211

Registros filtrados: 621
Registros a ejecutar: 3
Valor de max_i: 7532
Se inicia el servidor en Mac o Linux
Usando el chromedriver_path: /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver
7532. https://ctivitae.concytec.gob.pe/buscador-ui/#/ficha/ficha-renacyt?idInvestigador=204631
Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/renacyt/perfil/0_json
Se inicia el servidor en Mac o Linux
Usando el chromedriver_path: /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver
7533. https://ctivitae.concytec.gob.pe/buscador-ui/#/ficha/ficha-renacyt?idInvestigador=204693
Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/renacyt/perfil/0_json
Se inicia el servidor en Mac o Linux
Usando el chromedriver_path: /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver
7534. https://ctivitae.concytec.gob.pe/buscador-ui/#/ficha/ficha-renacyt?idInvestigador=204735
Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/renacyt/perfil/0_json
[H[2J[3JESTAMOS EN --->  linux
PROJECT_PATH:  /home/nahumfg/RenacytScraper
CHROME_DRIVER_PATH:  /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver

Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/renacyt/perfil/0_json
Registros originales: 7211

Registros filtrados: 618
Registros a ejecutar: 3
Valor de max_i: 7535
Se inicia el servidor en Mac o Linux
Usando el chromedriver_path: /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver
7535. https://ctivitae.concytec.gob.pe/buscador-ui/#/ficha/ficha-renacyt?idInvestigador=204802
Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/renacyt/perfil/0_json
Se inicia el servidor en Mac o Linux
Usando el chromedriver_path: /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver
7536. https://ctivitae.concytec.gob.pe/buscador-ui/#/ficha/ficha-renacyt?idInvestigador=204846
Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/renacyt/perfil/0_json
Se inicia el servidor en Mac o Linux
Usando el chromedriver_path: /home/nahumfg/RenacytScraper/chromedriver/linux/chromedriver
7537. https://ctivitae.concytec.gob.pe/buscador-ui/#/ficha/ficha-renacyt?idInvestigador=204926
Se eliminaron los archivos de la carpeta /home/nahumfg/RenacytScraper/originals/renacyt/perfil/0_json
>>>>>>> 29be084102095e8bbc001f2e99ad7d330f4a8fb1
[H[2J[3J